extension/cart/updateProductsInCart.js

```diff
@@ -152,19 +152,20 @@ function transformToUpdateItems (cartItems, magentoCart) {
  * @throws {Error}
  */
 function transformToUpdateItem (cartItem, cartItemMap) {
-  const magentoCartItem = cartItemMap[cartItem.cartItemId]
+  const cartItemId = cartItem.cartItemId ? cartItem.cartItemId : cartItem.CartItemId
```

Comment:
You could even shorten this by using the shorthand if operator ||:
 const cartItemId = cartItem.cartItemId || cartItem.CartItemId